boards/sim/sim/sim/src/sim_bringup.c

```diff
@@ -349,12 +349,12 @@ int sim_bringup(void)
 #endif
 
 #ifdef CONFIG_SIM_HCITTY
-  /* Register the Host Bluetooth network device via HCI socket */
+  /* Register the HCI TTY device via HCI socket */
 
-  ret = bthcitty_register(0);  /* Use HCI0 */
+  ret = hcitty_register("/dev/ttyHCI", 0);  /* Use HCI0 */
```

Comment:
This should have a major number? Im also not a huge fan of the interface taking the device string instead of a major number it creates a less strict interface which is then harder to develop applications on top of.

Reply:
> This should have a major number?

Since we just register one device now.

> Im also not a huge fan of the interface taking the device string instead of a major number it creates a less strict interface which is then harder to develop applications on top of.

But the application normally support the user config the device name anyway.

Reply:
Since this is constrained to `sim` I guess I am fine, but I would really like to see us have the device name be part of the driver definition and supply major(minor) to the interfaces, otherwise no one knows what is is "normally" known as.